tests/testthat/test-get_param.R

```diff
@@ -0,0 +1,48 @@
+test_that("can use configuration file via options", {
```

Comment:
I realized we had no tests for`get_param()` so I borrowed some from r2dii.utils and adapted them a bit.